Improve React,  Vuejs and Angular integration pages (#3948)
tinymce/tinymce-docs: modules/ROOT/partials/integrations/angular-load-tinymce-independently.adoc

+

> . There are two options for including {productname} into the application:

Reviewer: ```suggestion
. There are two options for including {productname} in an application:
```

.. Lazy load {productname} using the provided `TINYMCE_SCRIPT_SRC` dependency provider.
+
[source,ts]
----
import { EditorComponent, TINYMCE_SCRIPT_SRC } from '@tinymce/tinymce-angular';

@Component({
  /* ... */
  standalone: true,
  imports: [EditorComponent],
  providers: [
    { provide: TINYMCE_SCRIPT_SRC, useValue: '/tinymce/tinymce.min.js' }
  ]
})
----

> .. Add {productname} to the _global scripts_ tag via `+angular.json+`.

Reviewer 2: Just checking you want  _global scripts_ `italic`
```suggestion
.. Add {productname} to the _global scripts_ tag via `+angular.json+`.
```

Author: Updated!
